CONTRIBUTING.md

```diff
@@ -87,33 +87,33 @@ The success stories should be structured in the following way:
 
 You can read [Ahref's success story](https://ocaml.org/success-stories/peta-byte-scale-web-crawler) for an examplary success story.
 
-### <a name="content-user"></a> Add an Academic or Industrial User
+### <a name="content-user"></a><a name="academic-user"> Add an Academic or Industrial User</a>Industrial User<a name="industrial-user"></a>
 
 > Contribute to the [Academic Users](https://ocaml.org/academic-users) and [Industrial Users](https://ocaml.org/industrial-users).
 
 You can add a new academic user by creating a new Markdown file in [data/industrial_users/](data/industrial_users/). For instance: [cryptosense.md](data/industrial_users/en/cryptosense.md).
 
 You can add a new industrial user by creating a new Markdown file in [data/academic_institutions/](data/academic_institutions). For instance: [cornell.md](data/academic_institutions/en/cornell.md).
 
-### <a name="content-book"></a> Add a Book
+### <a name="content-book"></a><a name="books"> Add a Book</a>
 
 > Contribute to the [OCaml Books](https://ocaml.org/books).
 
 You can add a new OCaml book by creating a new Markdown file in [data/books/](data/books/). For instance: [ocaml-from-the-very-beginning.md](data/industrial_users/en/ocaml-from-the-very-beginning.md).
 
-### <a name="content-event"></a> Add an Event
+### <a name="content-event"></a><a name="event"> Add an Event</a>
 
 > Contribute to the [Community Events](https://ocaml.org/community).
```

Comment:
This is technically incorrectly named. Best to rename "event" here to "meetup" because that's all this is right now.

Reply:
@sabine  I moved the link from the event page to the community page as this is the location where I see the meetup is that correct?